velox/type/DecimalUtil.h

```diff
@@ -250,6 +250,21 @@ class DecimalUtil {
     return (unsignedSum >> 127);
   }
 
+  /// Adds two int128_t inputs, calculate the sum, and return the overflow.
+  /// It can be used to track the number of overflow when adding a batch of
+  /// input numbers. It takes lhs and rhs as input, and stores their sum in
+  /// result. overflow == 1 indicates upward overflow. overflow == -1
+  /// indicates downward overflow. overflow == 0 indicates no overflow.
+  /// When only one input is negative, adding them will never result in
```

Comment:
nit: Adding negative and non-negative numbers never overflows. Adding two negative or two positive numbers may overflow.